app/static/styles/womens_activism.css

```diff
@@ -2291,4 +2291,43 @@ p.interact, p.timePosted {
 
 .tag-remove{
     color: #630000;
+<<<<<<< Updated upstream
+=======
+}
+
+/* load-more button */
+#load-more{
+    margin: 5rem 0 0 0;
+    width: 40%;
+    max-width: 300px;
+    height: 50px;
+    -webkit-border-radius: 25px;
+    -moz-border-radius: 25px;
+    border-radius: 25px;
+    color: white;
+    background: #9857a3;
+    font-size: 22px;
+    border: 0;
+    outline: 0;
+    transition: background 0.2s ease;
+    font-weight: 600;
+}
+
+.loadMore {
+    display: flex;
+    justify-content: center;
+}
+
+#no-more-results {
+    margin: 5rem 0 0 0;
+    text-align: center;
+    font-size: 26px;
+}
+
+.request-description-text {
```

Comment:
Can you change this to `story-content-text`

Reply:
done